guides/source/documents.yaml

```diff
@@ -155,6 +155,10 @@
       name: Using Rails for API-only Applications
       url: api_app.html
       description: This guide explains how to effectively use Rails to develop a JSON API application.
+    -
+      name: Active Record and PostgreSQL
+      url: active_record_postgresql.html
```

Comment:
Can you make it `work_in_progress: true`?

Reply:
I've added a line to denote progress status.